src/system-health/tests/test_system_health.py

```diff
@@ -761,14 +761,20 @@ def test_post_system_status():
     print("post system status result:{}".format(result))
     assert result == "UP"
 
-@patch('health_checker.sysmonitor.Sysmonitor.publish_system_status', MagicMock())
-@patch('health_checker.sysmonitor.Sysmonitor.post_system_status', test_post_system_status())
+
 @patch('health_checker.sysmonitor.Sysmonitor.print_console_message', MagicMock())
+@patch('health_checker.sysmonitor.Sysmonitor.post_system_status', MagicMock())
 def test_publish_system_status():
     sysmon = Sysmonitor()
     sysmon.publish_system_status('UP')
-    result = swsscommon.SonicV2Connector.get(MockConnector, 0, "SYSTEM_READY|SYSTEM_STATE", 'Status')
-    assert result == "UP"
+    sysmon.publish_system_status('DOWN')
+    
```

Comment:
```suggestion
    sysmon.publish_system_status('UP')
    result = swsscommon.SonicV2Connector.get(MockConnector, 0, "SYSTEM_READY|SYSTEM_STATE", 'Status')
    assert result == 'UP'

    sysmon.publish_system_status('DOWN')
    result = swsscommon.SonicV2Connector.get(MockConnector, 0, "SYSTEM_READY|SYSTEM_STATE", 'Status')
    assert result == 'DOWN'
```
Would make it possible to make sure that the publish function has in fact worked, and the DB was updated as it should

Reply:
Taken care